src/Support/Field.php

```diff
@@ -204,6 +205,8 @@ protected function getResolver(): ?Closure
                 };
             }
 
+            $arguments[1] = (new AliasArguments($this->args(), $arguments[1]))->get();
```

Comment:
How about this:
- we add a new `protected function aliasArgs(array $args): array`
  ```php
  {
    return (new AliasArguments($this->args(), $arguments[1]))->get();
  }
  ```
- we add a new `protected function getArgs(array $args): array`
  ```php
  {
    return $this->aliasArgs($args);
  }
  ```
- we call it here like this: `$arguments[1] = $this->getArgs($arguments[1]);`

I see the benefit that:
- people can decide what to do with the args on their own
- by providing `aliasArgs` we make it easier to re-use if it people want to add additional steps within their `getArgs()`